test/suite/test_timestamp03.py

```diff
@@ -49,6 +49,7 @@ class test_timestamp03(wttest.WiredTigerTestCase, suite_subprocess):
     tablename = 'ts03_ts_nologged'
     tablename2 = 'ts03_nots_logged'
     tablename3 = 'ts03_ts_logged'
+    tablename4 = 'ts03_nots_nologged'
```

Comment:
Updated the tablenames as     
table_ts_log    
table_ts_nolog 
table_nots_log  
table_nots_nolog